packages/ra-ui-materialui/src/field/ReferenceArrayField.tsx

```diff
@@ -80,7 +103,22 @@ const useStyles = makeStyles(
     { name: 'RaReferenceArrayField' }
 );
 
-export const ReferenceArrayFieldView = ({
+type ReferenceArrayFieldClassKey = 'progress';
+
+interface ReferenceArrayFieldViewProps {
+    children: ReactElement;
+    classes?: Partial<ClassNameMap<ReferenceArrayFieldClassKey>>;
+    className: string;
+    data: RecordMap;
+    error?: any;
+    ids: Identifier[];
```

Comment:
not required (while loading, it's empty)

Reply:
Return props of [`useReferenceArrayFieldController`](https://github.com/marmelab/react-admin/blob/master/packages/ra-core/src/controller/field/useReferenceArrayFieldController.ts#L18-L19) which is injected into `ReferenceArrayFieldView` has the following interface:
```typescript
interface ReferenceArrayProps {
    ids: Identifier[];
    data: RecordMap;
    error?: any;
    loading: boolean;
    loaded: boolean;
    referenceBasePath: string;
}
```